EIPS/eip-867.md

```diff
@@ -0,0 +1,194 @@
+
+The ERP should link to pull requests for each client repository, and these pull requests should link back to the ERP and also contain its EIP preamble and summary. 
+
+Each PR associated with an ERP should consist of a single file (the SCO file) added to the client’s designated SCO directory.  No additional client code should be required.
+
+### Limitations of this EIP
+
+This EIP is focused on standardizing how recovery proposals can be formatted, to optimize readability and eliminate or minimize as much as possible the potential for mistakes or intentional abuses. 
+
+The following are considered out of scope from this EIP:
+- Which fund recovery proposals, if any, should be accepted for implementation. 
+- How common classes of recovery proposal plaintiff may organize ERPs representing a collective group of individual parties 
+ 
+
+## Rationale
+The primary consideration for the approach described above was to minimize the amount of risk associated with recovery actions that would otherwise not have a viable solution.  A secondary consideration was to standardize the format used in the proposals for recovery actions.
+
+First, including a verification script guarantees there the way in which the recovery actions were determined is unambiguous.  This does not mean that the recovery actions are necessarily correct, only that the logic used to determine them is fully specified and auditable.
+
+Second, requiring that the output of the verification script is directly interpretable by client programs minimizes the work necessary for each client to adopt a particular ERP.  It also reduces the risk that two clients will make different decisions about the implementation of a particular ERP.
+
+Third, action types are intentionally limited and inflexible, which reduces the likelihood of unintended consequences or maliciously constructed files affecting the blockchain state.  The format is easily extensible with new actions types if needed but that would require a separate EIP.
+
+
+## When to Write an ERP
+Before writing a new ERP, please review the ERPs that have already been written (draft, approved, and rejected) to determine if your recovery case is potentially covered already by an open ERP.  Even if it isn’t, similarities with other ERPs may help you to write a more effective ERP of find other people and organizations affected by your issue.  A single ERP for cases that have a common cause and solution will be more likely to find support than many smaller ERPs.  With that said, ERPs should be narrowly focused on a specific issue; combining cases that are unrelated into a single ERP will result in rejection.
+
+## Implementation
+A reference implementation has been written for the EthereumJ platform
```

Comment:
I would link the PR here for reference. The links below might break in future.

(Hint: pressing `Y` on a specific Github page generates permanent links.)